osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/services/DeviceManagementService.java

```diff
@@ -221,6 +222,7 @@ public Organisation findOrganisation(@Identification final String organisationId
             if (eventTypes != null && !eventTypes.isEmpty()) {
                 specification = specification.and(this.eventSpecifications.hasEventTypes(eventTypes));
             }
+            specification = handleDescription(description, descriptionStartsWith, specification);
```

Comment:
It would be more consistent to create method "this.eventSpecifications.hasDescription(...)".

Reply:
description is special, an thus handled in a special way